views/index.handlebars

```diff
@@ -6,6 +6,18 @@
   <script src="/bundle.js"></script>
   <link rel="stylesheet" href="https://code.cdn.mozilla.net/fonts/fira.css">
   <link rel="stylesheet" type="text/css" href="/main.css" />
+  {{#if shouldRenderAnalytics}}
+  <script>
+    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+    })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+
+    ga('create', '{{{trackerId}}}', 'auto');
+    ga('set', 'anonymizeIp', true);
+    ga('send', 'pageview');
+  </script>
```

Comment:
Not sure if this is exactly the same as the snippet in _views/download.handlebars_ above and if you want to refactor it out into a partial or something.

It may be as simple as moving the `<script>` snippet to _views/partials/analytics.handlebars_ and tweaking the following:

```js
const app = express();
app.engine('handlebars', exphbs({
  defaultLayout: 'main',
  partialsDir: 'views/partials/'
}));
app.set('view engine', 'handlebars');
```